packages/forms/src/elements/address/addressLookup.tsx

```diff
@@ -34,6 +34,7 @@ export type AddressProps = {
 	findAddressCancelledButton?: string;
 	onFindAddressCancelled?: () => void;
 	onValidatePostcode?: (isValid: boolean) => void | null;
+	onAddressChanging?: (isAddressChanging: boolean) => void | null;
```

Comment:
I'm not sure the purpose of the parameter is clear here. Why would an `onAddressChanging` event be fired if the address is not changing?